joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForTypesCreator.scala

```diff
@@ -171,8 +171,18 @@ trait AstForTypesCreator(implicit withSchemaValidation: ValidationMode) { this:
         .withChildren(singletonModifiers)
         .withChildren(fieldSingletonMemberNodes.map(_._2))
         .withChildren(singletonBodyAsts.toSeq)
+    val bodyMemberCall = astForMemberCall(
```

Comment:
This could probably be added during `RubyNodeCreator` to make AstCreator stuff neater